src/components/BookItem.js

```diff
@@ -1,16 +1,81 @@
-/* eslint-disable */
 import React from 'react';
-import { IoTrashBin } from 'react-icons/io5';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+const BookItem = (props) => {
+  const dispatch = useDispatch();
+
+  const {
+    title, author, id, category,
+  } = props;
+
+  const deleteBook = (id) => {
+    dispatch(removeBook(id));
+  };
 
-function BookItem(props) {
-  const { title, author, id } = props
   return (
-    <section className='displayBooks'>
-        <p className='book'> Book Title: {title} </p>
-        <p className='book'> Author's Name: {author} </p>
-        <button type='button' className='removeBtn' id={id}><IoTrashBin /></button>
-    </section>
-  )
-}
-
-export default BookItem;
\ No newline at end of file
+    <div className="displayBooks">
+      <div className="div-a">
+        <p className="book">
+          {' '}
+          {' '}
+          Category:
+          {' '}
+          {category}
+        </p>
+        <p className="book-title">
+          {' '}
+          {' '}
```

Comment:
- [ ] Please try to follow the Zeplin design guidelines as closely as possible so that your design is consistent with requirements. Use specified colors and font-sizes
- [ ] Please ensure that the background color of the card is `#fff`
- [ ] Please ensure to style the borders using the colours specified in the zeplin design
- [ ] Please ensure that the color of the text in the buttons is `#fff`
Kindly reference the attached images below 👇 

## Zeplin Design
![image](https://user-images.githubusercontent.com/46839548/202626231-ae41c81e-7e6a-4282-aeca-b74d0abe0fd5.png)

## Your Implementation
![image](https://user-images.githubusercontent.com/46839548/202625154-92916f70-af99-4ab7-8124-7ae7924001f1.png)

Reply:
Issue resolved!
Thank you for the constructive feedback.